synthtool/languages/python.py

```diff
@@ -55,7 +56,7 @@ def fix_pb2_headers(*, proto_root: str = "**/*_pb2.py") -> None:
     s.replace(
         proto_root,
         PB2_HEADER,
-        fr"\g<1>{LICENSE}\n\n\g<2>",  # change order to avoid stacking replacements
+        rf"\g<1>{LICENSE}\n\n\g<2>",  # change order to avoid stacking replacements
```

Comment:
Does changing the order change behavior?

Reply:
Possibly. I'll revert this change.